synced_collections/data_types/synced_dict.py

```diff
@@ -44,7 +44,7 @@ class SyncedDict(SyncedCollection, MutableMapping):
     there are important distinctions to remember. In particular, because
     operations are reflected as changes to an underlying backend, copying (even
     deep copying) a :class:`SyncedDict` instance may exhibit unexpected
-    behavior. If a true copy is required, you should use the `_to_base()`
+    behavior. If a true copy is required, you should use the :meth:`~._to_base`
```

Comment:
@vyasr Just verifying -- do we want to recommend `_to_base` or the call operator? The concrete classes recommend the call operator but `SyncedDict` and `SyncedList` recommend using `_to_base` directly. The difference is that the call operator does a `_load` first.

Reply:
Good catch. Definitely want to use the call operator. These docs have never been published, so this was probably missed when we originally reviewed and then we never updated it.

Reply:
Added docs of the call operator so that we can link to it.